app/assets/stylesheets/pages/_index.scss

```diff
@@ -5,3 +5,38 @@
 body {
   background-color: white;
 }
+
+.banner {
+height: 458px;
```

Comment:
@vitorreis  Commenting here as example: Why using a fixed height here? I believe this would make the design not responsive to different monitor heights.
Suggestion: remove this line